sympy/vector/deloperator.py

```diff
@@ -13,20 +9,11 @@ class Del(Basic):
     mathematical expressions as the 'nabla' symbol.
     """
 
-    def __new__(cls, system):
-        if not isinstance(system, CoordSysCartesian):
-            raise TypeError("system should be a CoordSysCartesian")
-        obj = super(Del, cls).__new__(cls, system)
-        obj._x, obj._y, obj._z = system.x, system.y, system.z
-        obj._i, obj._j, obj._k = system.i, system.j, system.k
-        obj._system = system
-        obj._name = system.__str__() + ".delop"
+    def __new__(cls, ):
+        obj = super(Del, cls).__new__(cls)
+        obj._name = "delop"
```

Comment:
We don't know how the users are using it. We should put deprecation warnings everywhere, maybe some users have written scripts that call `Del` directly, they need a warning to change the code. Usually deprecation warnings stay for 1-2 years, then the deprecated code is completely removed.